pkg/config/config_windows.go

```diff
@@ -0,0 +1,24 @@
+// +build windows
+
+/*
+Copyright The containerd Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+// DefaultConfig returns default configurations of cri plugin.
+func DefaultConfig() PluginConfig {
+	return PluginConfig{}
```

Comment:
i was only expecting the paths to be different for the most part in things like configs... which would require a two different sets of paths approach based on windows/linux ... not two copies of config. For example, StreamServerAddress: "127.0.0.1", local host the same on windows? As another example, sandbox image, is it multi arch? If so should be the same image. Still another, docker registry should be the same right? 

Put another way... are we just forking entire sets of functions if there might be a difference of one path or one feature in a function, or are we forking because we know it will be very unique and no simple separation of paths and tools used could be determined. What's the plan?

Reply:
It works in place :). Mostly haha. @Random-Liu - Should you fill this out with the defaults that matter from our fork for the checkin or do you want to do that as a followup?

Reply:
@mikebrow Here is the diff:
```diff
< // +build !windows
---
> // +build windows
30,32c30,32
< 			NetworkPluginBinDir:       "/opt/cni/bin",
< 			NetworkPluginConfDir:      "/etc/cni/net.d",
< 			NetworkPluginMaxConfNum:   1, // only one CNI plugin config file will be loaded
---
> 			NetworkPluginBinDir:       "C:\\Program Files\\containerd\\cni\\bin",
> 			NetworkPluginConfDir:      "C:\\Program Files\\containerd\\cni\\conf",
> 			NetworkPluginMaxConfNum:   1,
37c37
< 			DefaultRuntimeName: "runc",
---
> 			DefaultRuntimeName: "runhcs",
40,41c40,41
< 				"runc": {
< 					Type: "io.containerd.runc.v1",
---
> 				"runhcs": {
> 					Type: "io.containerd.runhcs.v1",
49d48
< 		EnableSelinux:       false,
55c54
< 		SandboxImage:            "k8s.gcr.io/pause:3.1",
---
> 		SandboxImage:            "mcr.microsoft.com/k8s/core/pause:1.0.0",
57d55
< 		SystemdCgroup:           false,
67d64
< 		DisableProcMount:       false,
```

Let's see whether we can further clean this up as part of https://github.com/containerd/cri/issues/1245.

Reply:
Added https://github.com/containerd/cri/issues/1245#issuecomment-527662192

Reply:
@Random-Liu - Knowing there is a future here can we not add "runhcs" and use "runhcs-wcow-process". It makes more sense and is more descriptive once the other two: "runhcs-wcow-hypervisor" and "runhcs-lcow" are added

Reply:
Sure. Will do in the next PR when actually adding the default config.